arch/x86/zefi/README.txt

```diff
@@ -0,0 +1,92 @@
+Operation
```

Comment:
Why not just 'efi' as a directory name?

Reply:
Mostly because "EFI" is a whole environment we might want to support in different ways (e.g. runtime access to EFI services, etc..., or as @stephanosio  points out by linking Zephyr into a EFI image directly).  This is a standalone tool so I gave it a standalone name, basically.